make/autoconf/flags-cflags.m4

```diff
@@ -664,9 +664,7 @@ AC_DEFUN([FLAGS_SETUP_CFLAGS_CPU_DEP],
       -D$FLAGS_CPU_LEGACY"
 
   if test "x$FLAGS_CPU_BITS" = x64; then
-    # -D_LP64=1 is only set on linux and mac. Setting on windows causes diff in
-    # unpack200.exe.
-    if test "x$FLAGS_OS" = xlinux || test "x$FLAGS_OS" = xmacosx; then
+    if test "x$FLAGS_OS" = xlinux || test "x$FLAGS_OS" = xmacosx || test "x$FLAGS_OS" = xwindows; then
```

Comment:
At this point, you're almost testing for all supported OSes.  :) I can only think of AIX that does not match this if clause. I think it would be better to just remove the if and always define _LP64=1 on 64-bit platforms. 

@simonis Would that be OK for AIX?

Reply:
Thanks for the suggestion.

Note the code that adds _LP64 for the JVM (below):

```
    if test "x$FLAGS_OS" != xaix; then
      # xlc on AIX defines _LP64=1 by default and issues a warning if we redefine it.
      $1_DEFINES_CPU_JVM="${$1_DEFINES_CPU_JVM} -D_LP64=1"
    fi
```

So, it seems xlc/aix explicitly does _not_ want this macro defined?

I think we could reuse that if-block to add `-D_LP64=` to both `1_DEFINES_CPU_JDK`, and `1_DEFINES_CPU_JVM` though, and remove the first one that checks for linux/mac/windows.

Reply:
Yes, that sounds good. I did not notice this (still not used to github reviews, which I think has too little context by default).

Reply:
Ok, will do. (FWIW, you can expand the context of the diff with the arrow buttons on the left side of the view. Above or below the line numbers)

Reply:
(Yes, I know.  I just didn't think that doing so would reveal anything about AIX. I just wish I had gotten a bit more context automatically.)

Reply:
Sorry, but I'm neither workin on AIX any more nor do I have access to an AIX machine :)

Pulling in @mdoerr or @tstuefe